xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
Ah, this is what you mean:There is a general resync on Video at the time the Audio packet with the change is processed (and this is 0.5 secs before the video has to change: starttime - cache, ok, I'll revert my things and will try to supress / postpone the video resync to a later time.